ApollosProject/apollos-prototype
# Moves liked content feeds to @apollosproject/ui-connected

## DESCRIPTION
<img src="https://user-images.githubusercontent.com/2053547/72636628-7061f180-392d-11ea-8189-cdeabcbe6618.png" width="30%" /><img src="https://user-images.githubusercontent.com/2053547/72636647-7bb51d00-392d-11ea-92a9-b77919328e5b.png" width="30%" />

### What does this PR do, or why is it needed?
This PR moves `RecentlyLikedList` and `RecentlyLikedTileList` to the `ui-connected` package. It also renames those packages and does a very light handed refactor to update them to follow our desired conventions. Lastly this copies over/migrates any necessary `utils` to the `ui-connected` package.

### What design trade-offs/decisions were made?
These components need to import `ContentCardConnected` and `HorizontalContentCardConnected`. Those cards will be moved to `ui-connected` soon but for now they are just imported from `apolloschurchapp` so that nothing breaks and things can be tested.

### How do I test this PR?
1. open the app
2. tap on connect, see "recently liked" section
3. tap on content, it works.
4. tap back and tap "view all", loads feed.
5. tap back anywhere and test that liking content shows up in the feeds above.
6. 💥 

### What automated tests did you write?
same coverage as before.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
